Makefile

```diff
@@ -91,6 +93,28 @@ connect: _need_hosttool _need_ttkmd
 ssh:
 	ssh -F /dev/null -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o NoHostAuthenticationForLocalhost=yes -o User=debian -p2222 localhost
 
+# Launch tmux with a 2x2 grid and connect to each l2cpu in each
+connect_all:
+# Kill any existing sessions named connect_all
+	tmux has-session -t connect_all 2>/dev/null && tmux kill-session -t connect_all || true
+	tmux new-session -d -s connect_all 'bash'
+	tmux split-window -h -t connect_all:0
+	tmux split-window -v -t connect_all:0.0
+	tmux split-window -v -t connect_all:0.1
+	tmux select-layout -t connect_all:0 tiled
+	tmux send-keys -t connect_all:0.0 './console/tt-bh-linux --l2cpu 0' C-m
+	tmux send-keys -t connect_all:0.1 './console/tt-bh-linux --l2cpu 1' C-m
+	tmux send-keys -t connect_all:0.2 './console/tt-bh-linux --l2cpu 2' C-m
+	tmux send-keys -t connect_all:0.3 './console/tt-bh-linux --l2cpu 3' C-m
```

Comment:
I slightly prefer doing the command as part of the split-window, since the `C-a x` then closes the pane. But this is works too.